packages/models-library/src/models_library/aiodocker_api.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+from pydantic import Field, validator
+
+from .generated_models.docker_rest_api import ContainerSpec, ServiceSpec, TaskSpec
+
+
+def to_snake_case(string: str) -> str:
```

Comment:
MINOR: could gather all these converters in a single place, e.g. under ``models_library.utils``